src/server.h

```diff
@@ -205,6 +205,7 @@ class Server : public con::PeerHandler, public MapEventReceiver,
 
 	// This is run by ServerThread and does the actual processing
 	void AsyncRunStep(float dtime, bool initial_step = false);
+	// Receive all incoming packets and wait for at least `timeout` seconds.
```

Comment:
```suggestion
	/// Receive and process all incoming packets. Sleep if the time goal isn't met.
	/// @param timeout minimum time to take [s]
```

Reply:
Hah, "timeout" is pretty much the opposite of "minimum time", the name is really misleading now. Renamed to "min_time".